headers/functions/overlay29.h

```diff
@@ -867,5 +867,8 @@ void DisplayMessageInternal(int message_id, bool wait_for_input, struct portrait
 void OpenMenu(undefined param_1, undefined param_2, bool open_bag);
 int OthersMenuLoop(void);
 undefined OthersMenu(void);
+int GetAiUseItemProbability(struct entity* item_consumer, struct item* used_item, int unknown);
```

Comment:
> struct item* used_item

I think `used_item` is a bit misleading. It seems to imply that the item has already been used. Why not just name it `item`?

> int unknown

I think that parameter should either be documented as a bitfield or specified as `undefined`, since saying that it's an int is not 100% true.